zarr/storage.py

```diff
@@ -1883,3 +1883,121 @@ def __delitem__(self, key):
         with self._mutex:
             self._invalidate_keys()
             self._invalidate_value(key)
+
+
+class ChunkCache(MutableMapping):
```

Comment:
Maybe should rename this to "LRUChunkCache" just to be explicit about caching strategy.